resources/views/admin/category/categories.blade.php~[SELS-TASK] Implemented Create Category in Admin Panel

```diff
@@ -0,0 +1,56 @@
+@extends('layout')
+
+@section('content')
+<div class="row">
```

Comment:
what is this file?

Reply:
I think this is a merge-conflict generated file. I'll get rid of it.